tsconfig.json

```diff
@@ -12,14 +12,26 @@
     "rootDir": "src",
     "outDir": "dist",
     "lib": ["ES2021"],
+    // Customize or add to these paths as needed
     "baseUrl": "src",
     "paths": {
-      "$components/*": ["views/components/components.ts"],
-      "$modals/*": ["views/modals"],
-      "$messages/*": ["views/messages"],
-      "$home-tab/*": ["views/home-tab"],
-      "$emojis/*": ["utilities/emojis.ts"],
-      "$constants/*": ["utilities/constants.ts"]
+      "$actions": ["actions"],
```

Comment:
I'm curious how this works. The first line is access to the Actions index file and the second is if you need access to only a specific action file?